app/layout.tsx

```diff
@@ -0,0 +1,60 @@
+import { Footer } from 'components/common/Footer';
+import './globals.css';
+import Header from 'components/common/Header';
+import { Metadata } from 'next';
+
+const description = 'Track all transactions on Rango Exchange';
+const socialTitle = 'Rango Exchange Explorer';
+const BASE_URL = 'https://scan.rango.exchange';
+const APP_NAME = 'Rango Exchange Explorer';
+
+export const metadata: Metadata = {
+  title: 'Rango Exchange Explorer',
+  description: description,
+  keywords:
+    'Rango Exchange, Rango dApp, Multi-chain DEX aggregator, Cross-Chain Swap, binance bridge, 1inch, crypto API, Metamask',
+  icons: {
+    icon: '/favicon.svg',
```

Comment:
move the metadata file to the app root directory.
use app router conventions: https://nextjs.org/docs/app/api-reference/file-conventions/metadata/app-icons

Reply:
Tnx done in https://github.com/rango-exchange/explorer/pull/32/commits/9786fd1a66c6d79fbb8dfb4b69203547cc46a9b5